### PR TITLE
Feature/update readme

### PR DESCRIPTION
**Pull request by GAndriyS**
update repo readme file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,8 @@
-# munchkin_cardbase
\ No newline at end of file
+### To run dev envoirement follow instructions below:
```
Comment (vira-lytvyn) [inline README.md:1]: Do you mean that these steps describe how to run up app locally?

**Review by vira-lytvyn — CHANGES_REQUESTED** [review]
Please add a license file to your project. This is compulsory for a project(open source inclusively) to have a license.

**Comment by vira-lytvyn** [comment]
*just a note: it is nice to mention what exactly were modified/deleted/implemented in a specific commit description in order to simplify history processing.
For instance, for this commit, you may use something like 'Add an instruction to readme about how to run the  app locally.'

**Review by vira-lytvyn — COMMENTED** [review]
I've left some comment and requested a change.
Please process these before merging the current PR.

File: README.md (hunk 1 above)
Comment (GAndriyS) [reply README.md:1]: yup

**Comment by GAndriyS** [comment]
I've added license file to repo. Check it out pls,

File: README.md (hunk 1 above)
Comment (vira-lytvyn) [reply README.md:1]: then you have to write this) from your description people may think that this is an instruction on how to set up the development environment
with the documentation you need to be precise and very careful - you don't know who and when will use it and what experience he has

**Review by vira-lytvyn — APPROVED** [review]

**Review by vira-lytvyn — APPROVED** [review]
Greate!

**Pull request merged**